src/api/projects.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+import {User} from '../db/models/user';
+import {Project} from '../db/models/project';
+import Permission from '../util/permission';
+
+/**
+ * Create project under user's organization
+ * @param  {object} event
+ * @param  {object} context
+ * @param  {object} callback
+ * @return {function} done
+ */
+export const create = (event, context, callback) => {
+  const done = (error, res) => callback(null, {
+    statusCode: error ? '403' : '201',
+    body: error ? JSON.stringify(error) : JSON.stringify(res),
+    headers: {
+      'Content-Type': 'application/json',
+      'Access-Control-Allow-Origin': '*',
+    },
+  });
+
+  const name = JSON.parse(event.body).name;
+  if (name == null || name.trim() == '') {
+    return done({
+      message: `Request doesn't contain a valid object`,
+    });
+  }
+
+  const uuid = event.requestContext.authorizer.principalId;
+  Permission.hasPermission(uuid, {realm: 'project', action: 'create'})
```

Comment:
@singhabahu cant' we simplify this more ?, l see some readability issues.

Reply:
any suggestions to improve readability?, this is used to check permission of the user against requested action. any ideas @SavidyaSathyani ?